coffee_cmd/src/cmd.rs

```diff
@@ -1,5 +1,8 @@
 //! Coffee command line arguments definition.
 use clap::{Parser, Subcommand};
+use coffee_lib::error;
+use coffee_lib::errors::CoffeeError;
+use std::fmt::Display;
```

Comment:
look at how we organize the import, we follow the following rules

```
std import

external dependencies import

coffee importa

crate import
```

But this can be addressed in a followup PR

Reply:
Thank you for pointing that out. I'll address that asap!

Reply:
I've addressed this in #276.